src/Nethermind/Nethermind.Api/NethermindApi.cs

```diff
@@ -226,6 +226,7 @@ public ISealEngine SealEngine
         public CensorshipDetector CensorshipDetector { get; set; } = null!;
         public IWallet? Wallet { get; set; }
         public IBadBlockStore? BadBlocksStore { get; set; }
+        public BlockStore? BlockStore { get; set; }
```

Comment:
Shouldn't it be `IBlockStore` in the API?

Reply:
Moved to ISyncPointers.